meta-mender-core/classes/mender-setup.bbclass

```diff
@@ -160,6 +160,18 @@ MENDER_UBOOT_PRE_SETUP_COMMANDS_DEFAULT = ""
 MENDER_UBOOT_POST_SETUP_COMMANDS ??= "${MENDER_UBOOT_POST_SETUP_COMMANDS_DEFAULT}"
 MENDER_UBOOT_POST_SETUP_COMMANDS_DEFAULT = ""
 
+# All the allowed mender configuration variables
+MENDER_CONFIGURATION_VARS ?= "\
+    RootfsPartA \
+    RootfsPartB \
+    InventoryPollIntervalSeconds \
+    RetryPollIntervalSeconds \
+    ArtifactVerifyKey \
+    ServerCertificate \
+    ServerURL \
+    UpdatePollIntervalSeconds"
```

Comment:
One unified place for storing this info. Don't know if this is your preference, @kacf.